components/MyPlants.js

```diff
@@ -39,12 +51,14 @@ export default function MyPlants({
       ) : (
         <StyledList>
           {myPlants.map((plant) => {
+            const isUserPlant = getPlantInfoById(plant.id)?.isUserPlant;
+            const isMyPlant = getPlantInfoById(plant.id)?.isMyPlant;
```

Comment:
you can make this shorter and more efficient:
```suggestion
            const plantInfo = getPlantInfoById(plant.id)
            const {isUserPlant, isMyPlant} = plantInfo
```
You can skip the variable "plantInfo" and destructure the return value of getPlantInfoById directly in the first line. You can decide what you find more pleasing.